### ISSUE TITLE
module search gives help when it shouldn't

### ISSUE THREAD

**Issue by digininja** [lanmaster53/recon-ng#45]
When the search is used correctly, it gives the reply to say nothing is found, but then still dumps out the usage information:

```
[recon-ng][default] > modules search email
[*] Searching installed modules for 'email'...
[!] No modules found.
Searches installed modules

Usage: modules search [<regex>]
```

**Comment by lanmaster53** [comment]
I believe I have it set up where any time the user doesn't get search results, it reminds the user how the search interface works just in case they forgot to use regex.

**Comment by digininja** [comment]
Fair enough if it is deliberate, I just didn't expect help info when it
looked like I'd used the function correctly.

**Comment by lanmaster53** [comment]
You're right though. I plan to look at the logic and figure out of there is a less jarring way to achieve the same goal. Thanks for the feedback.

**Comment by digininja** [comment]
Want this closing or left open as a reminder?

**Comment by lanmaster53** [comment]
Leave it open for now. Thanks.
